compiler/circle-quantizer/src/CircleQuantizer.cpp

```diff
@@ -117,6 +118,46 @@ LayerParams read_layer_params(std::string &filename)
   return p;
 }
 
+LayerParamsSet read_layer_params_set(std::string &filename)
```

Comment:
```suggestion
LayerParamsSet read_layer_params_set(const std::string &filename)
```

Reply:
FYI need to add `const` also to existing `read_layer_params()`